backend/datasource.go

```diff
@@ -91,15 +91,24 @@ func (ds *CassandraDatasource) MetricQuery(tsdbReq *datasource.DatasourceRequest
 
 		var created_at time.Time
 		var value float64
+
+		ds.logger.Debug(fmt.Sprintf("filtering: %s\n", queryData.Get("filtering").MustString()))
+
+		allowFiltering := func() string { if queryData.Get("filtering").MustString() == "Disallow" { return "" } else { return "ALLOW FILTERING" } }()	
```

Comment:
1. It's better to use conventional True or False than "Disallow"
2. If you would return " ALLOW FILTERING" (notice the space before Allow), you would omit a redundant space in the line 100